core/src/test/java/io/atomix/core/AtomixTest.java

```diff
@@ -230,6 +231,7 @@ public void onEvent(ClusterEvent event) {
       try {
         queue.put(event);
       } catch (InterruptedException e) {
+        Throwables.propagate(e);
```

Comment:
Isn’t this deprecated now? Not sure what’s supposed to replace it though.

Reply:
Yep, in this cause I would probably just interrupt the thread again, but instead copied code which is like this in about three places..

Reply:
updated.